woocommerce-services.php

```diff
@@ -249,6 +256,18 @@ class WC_Connect_Loader {
 
 		public static function plugin_deactivation() {
 			wp_clear_scheduled_hook( 'wc_connect_fetch_service_schemas' );
+
+			/**
+			 * When we deactivate the plugin after wcshipping_migration_state has started,
+			 * that means the migration is done. We can mark it as completed before we deactivate the plugin.
+			 */
+			$migration_state = WC_Connect_Options::get_option( 'wcshipping_migration_state' );
+			if ( ! class_exists( WC_Connect_WCST_To_WCShipping_Migration_State_Enum::class ) ) {
```

Comment:
Do we need this extra check? I imagine `require_once` already does something like this internally.

Reply:
No we don't. I took the existing pattern/practice in the WCS&T code where the import starts with a `if ( ! class_exists()` check.

It is useful when we have 2 different file paths with the same class name (collision). 

Here, we don't have this problem because `WC_Connect_WCST_To_WCShipping_Migration_State_Enum` is a class I just added to WCS&T, there is no chance of collision. `require_once` suffices.

Reply:
Updated in d14b131c